src/state-manager/TransactionQueue.ts

```diff
@@ -2182,6 +2182,19 @@ class TransactionQueue {
           //set the nodes that are in the executionGroup.
           //This is needed so that consensus will expect less nodes to be voting
           const unRankedExecutionGroup = homeShardData.homeNodes[0].consensusNodeForOurNodeFull.slice()
+
+          // here we  reject the transaction if the executionGroup.length is lesser than minimumConsensusGroup size, so that we dont have to go ahead with the remaining processing and hereby maintaining the safe node count for consensus
+          if (txQueueEntry.executionGroup.length < this.config.sharding.minNodesPerConsensusGroup) {
```

Comment:
@abdulazeem-tk4vr is it possible that execution group length is greater than `minNodesPerConsensusGroup`? If so, is that fine in the premise of this ticket?

Reply:
yes that is fine.